Cpp/source/DrawingEngine/FrameBuffer/FrameBuffer.cpp

```diff
@@ -0,0 +1,40 @@
+#include "FrameBuffer.hpp"
+#include <bits/stdc++.h> 
+#include <stdexcept>
+
+void FrameBuffer::set_size(int buf_size)
```

Comment:
this should be done in constructor, in current state after creating frame buffer it is in "invalid state"

Reply:
done